segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/operations/OperationSerializer.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.logs.operations;
+
+import io.pravega.common.io.serialization.VersionedSerializer;
+import io.pravega.segmentstore.server.logs.Serializer;
+
+/**
+ * Operation Serializer.
+ */
+public class OperationSerializer extends VersionedSerializer.MultiType<Operation> implements Serializer<Operation> {
+    public static final OperationSerializer DEFAULT = new OperationSerializer();
+
+    @Override
+    protected void declareSerializers(Builder b) {
```

Comment:
This particular "Builder" is a nested type of `VersionedSerializer.MultiType`, so it's only accessible here because this class subclasses `MultiType`.